vllm/entrypoints/openai/protocol.py

```diff
@@ -52,9 +52,60 @@ class UsageInfo(BaseModel):
     completion_tokens: Optional[int] = 0
 
 
+class FunctionCall(BaseModel):
```

Comment:
Probably should be cleaner to use the same class name as OAI both  for `FunctionCall` and `ToolCallsMessage`

https://github.com/openai/openai-python/blob/main/src/openai/types/chat/chat_completion_message_tool_call.py